pkg/addon/versioned_registry.go

```diff
@@ -212,3 +231,22 @@ func chooseVersion(specifiedVersion string, versions []*repo.ChartVersion) (*rep
 	}
 	return addonVersion, availableVersions
 }
+
+// LoadSystemRequirements load the system version requirements from the addon's meta file
+func LoadSystemRequirements(requirements string) *SystemRequirements {
+	if len(requirements) == 0 {
+		return nil
+	}
+	patternReq := `vela(.*\d+\.\d+\.\d+);\s?kubernetes(.*\d+\.\d+\.\d+)`
```

Comment:
please define this string as a const str.

Reply:
ok